src/index.ts

```diff
@@ -1,17 +1,16 @@
 import * as core from '@actions/core';
 import * as github from '@actions/github';
 
-const getVersion = async (version: string): Promise<Version> => {
-    const numbers = version.split('.');
-    console.log("numbers:", numbers);
+const getVersion = async (version: Array<any>): Promise<Version> => {

```

Comment:
```suggestion
const getVersion = async (version: RegExpMatchArray): Promise<Version> => {
```

This would avoid having `any` in the types here.

Reply:
Good catch! updated it!